config/sync/core.extension.yml

```diff
@@ -28,6 +28,7 @@ module:
   datetime: 0
   datetime_range: 0
   diff: 0
+  dynamic_entity_reference: 0
```

Comment:
Didn't expect to see this module for scheduled transitions. Is it a dependency?

Reply:
Yup!